tox.ini

```diff
@@ -11,6 +11,7 @@ deps =
     coverage
     botocore
     requests
+    aiohttp
```

Comment:
Did you test running $tox when you have python2.7 or 3.4 installed? The general development workflow for non-language-version-specific features is that contributors should be able to just run $tox against all version of interpreter installed so they know the new changes don't break for any specific version. The aiohttp test would be better be broken into a new command to run with only py3.5 and later.

Reply:
Ahh no I just ran the py3.5 test manually, yeah makes sense as it would fail on anything less than 3.5